app.json

```diff
@@ -18,7 +18,9 @@
             "generator": "secret"
         }
     },
-    "buildpacks": {
-            "url": "heroku/python"
-    }
-}
+    "buildpacks": [
+      {
+        "url": "heroku/python"
+      }
+    ]
+  }
```

Comment:
If you don't know what POSIX is, this wikipedia article might come handy: https://en.wikipedia.org/wiki/POSIX